### PR TITLE
Header

### PR DESCRIPTION
**Pull request by Apatakiti**
- Add the header

### CODE REVIEW

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @Ademola-coding , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors

## Required Changes ♻️
_Check the comments under the review._

- **Kindly add a proper description to your PR before submitting it for a rereview.**
- Consider using the same font size throughout your application as in the Figma template.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: files/style.css [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: #1d2379;
+  position: absolute;
+  background-image: url("Ellipse 18.png");
+  background-repeat: no-repeat;
```
Comment (Ginohmk) [inline files/style.css:11]: **Note** Please do not close a PR except if it is approved.

- Please use the proper font size and font-family as it is in the Figma design, Currently, your font family is different from the Figma design 
Implement changes and request a review with the right PR

File: files/style.css [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+}
+
+.heading {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+}
```
Comment (Ginohmk) [inline files/style.css:18]: - Please use a similar social media icon to the Figma design as yours is different from the Figma design this is to help you follow the details of a design properly. And also change the color of the horizontal line to be similar to that of the Figma design. `background: #646369;`

##### Your design 
Screenshot from 2022-07-01 19-29-00


##### Figma design 
Screenshot from 2022-07-01 19-30-02

File: files/style.css [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+
+.logo {
+  position: relative;
+  align-self: flex-start;
+  color: #fff;
+  font-weight: 600;
```
Comment (Ginohmk) [inline files/style.css:24]: - Please properly align the header logo and the hamburger menu as it is not aligned like that of the figma design.

##### Your design
Screenshot from 2022-07-01 19-22-06

##### Figma design
Screenshot from 2022-07-01 19-32-03

**Review by Ginohmk — COMMENTED** [review]
Hi @Ademola-coding , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

keep-trying-youre-almost-there

 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ginohmk — COMMENTED** [review]
Hi @Ademola-coding  

# STATUS: APPROVED 🟢

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**